### DISCUSSION TITLE
Can I use Authentik with react.JS frontEnd & Express backend

### DISCUSSION THREAD

**Discussion by mustanish** [goauthentik/authentik#3848] [General]
My requirement is I want to use Authentik as a single sign-on service for all the services we are using. we have a react frontend and express backend for managing a few stuff. We are also using a few third-party software that has links on react frontend, what we want is if a user is authorized once using react APP, the user should not be asked to authorize again once they move to other third-party software by clicking on links.  Suggest any flow

**Comment by ali-whizz** [comment]
did you able to find the solution if yes please share here i have the same use case scenario
